fluentd-forward/Makefile

```diff
@@ -0,0 +1,72 @@
+VERSION=0.0.1
```

Comment:
Please add a `release` target to the Makefile similar to https://github.com/gravitational/teleport-plugins/pull/248.